src/main/java/org/beanpod/switchboard/entity/EncoderEntity.java

```diff
@@ -29,7 +29,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @JsonIgnoreProperties(value = {"hibernateLazyIntializer", "handler"})
-public class EncoderEntity {
+public class EncoderEntity implements DummyInterface {
```

Comment:
nice idea!

Reply:
@MrJCipherButtles this interface serves a single purpose which is avoiding duplicating maintainStatusField method. If DecoderEntity and EncoderEntity classes don't implement a common interface, then we would have to write one method for each although the logic is exactly the same. I've added a comment on the top of the interface to give clear idea what this interface is for.